backend/CircleForms/Contracts/ContractModels/Response/Users/UserContract.cs

```diff
@@ -12,5 +13,5 @@ public class UserContract
     public string Discord { get; set; }
 
     [JsonProperty("osu")]
-    public Dictionary<string, object> Osu { get; set; }
+    public OsuUser Osu { get; set; }

```

Comment:
`OsuUser` contains fields that will not be stored in the database according to `BsonIgnore`, we need another contract